pyproject.toml

```diff
@@ -30,7 +30,8 @@ include = [
     "templates/*.txt",
     "templates/*.html",
     "templates/*.css",
-    "templates/*.md"
+    "templates/*.md",
+    {path = 'tests/*', format = 'sdist'},
 ]
```

Comment:
@dvzrv Looks like you cant mix types like this. 

In my testing the fix is rather easy

```
include = [
    {path = 'templates/*.txt'},
    {path = 'templates/*.html'},
    {path = 'templates/*.css'},
    {path = 'templates/*.md'},
    {path = 'tests/*', format = 'sdist'}
]
```

If you update this pr with that im fairly sure it will pass

Reply:
Hmm, why does that fail though? To my knowledge [it is valid TOML to have a trailing comma on all members of an array](https://toml.io/en/v1.0.0#array) (in Javascript this would break I think).

Additionally, I have been using it like that in several projects without any issue. :confused:

Reply:
Ah, I guess the [toml](https://github.com/uiri/toml) library is at fault here by [only supporting TOML 0.5.0](https://raw.githubusercontent.com/uiri/toml/2846c42e85f56090da815455a1cb307ed6d6acb1/README.rst). This is probably one of the reasons why projects across the board switch to [tomli](https://github.com/hukkin/tomli), which in turn [supports TOML 1.0.0](https://github.com/hukkin/tomli/blob/a103b0798e55c7d06c57c176895422e9cf403050/README.md#intro).

If you want I can of course change the array, but going forward it will make more sense to support current specifications of TOML I guess.

Reply:
Ah lol, I need to change the type of the array member as well. I didn't get that (sorry, there was no coffee....)

Reply:
As far as Tomli we do have is as an optional dep (if users want to use file based config). I don't think it's our code base that is exploding here. I have not looked too closely but I think it's pytest that gets upset.

Not 100% I'm looking at this on mobile

Reply:
I think the trailing comma will be okay (not 100% sure either) with python-toml. I think the underlying issue was the different member types in that array.